dspace-api/src/main/java/org/dspace/core/Email.java

```diff
@@ -455,21 +488,30 @@ public static Email getEmail(String emailFile)
      * @param args the command line arguments given
      */
     public static void main(String[] args) {
-        ConfigurationService config = DSpaceServicesFactory.getInstance().getConfigurationService();
+        ConfigurationService config
+                = DSpaceServicesFactory.getInstance().getConfigurationService();
         String to = config.getProperty("mail.admin");
         String subject = "DSpace test email";
         String server = config.getProperty("mail.server");
         String url = config.getProperty("dspace.url");
-        Email e = new Email();
-        e.setSubject(subject);
-        e.addRecipient(to);
-        e.content = "This is a test email sent from DSpace: " + url;
-        System.out.println("\nAbout to send test email:");
-        System.out.println(" - To: " + to);
-        System.out.println(" - Subject: " + subject);
-        System.out.println(" - Server: " + server);
-        boolean disabled = config.getBooleanProperty("mail.server.disabled", false);
+        Email message;
         try {
+            if (args.length <= 0) {
```

Comment:
If I run the test-email script without arguments, I get the following exception:
```
Exception: Unable to find resource 'null'
org.apache.velocity.exception.ResourceNotFoundException: Unable to find resource 'null'
	at org.apache.velocity.runtime.resource.ResourceManagerImpl.loadResource(ResourceManagerImpl.java:462)
	at org.apache.velocity.runtime.resource.ResourceManagerImpl.getResource(ResourceManagerImpl.java:343)
	at org.apache.velocity.runtime.RuntimeInstance.getTemplate(RuntimeInstance.java:1554)
	at org.apache.velocity.runtime.RuntimeInstance.getTemplate(RuntimeInstance.java:1534)
	at org.apache.velocity.app.VelocityEngine.getTemplate(VelocityEngine.java:301)
	at org.dspace.core.Email.send(Email.java:331)
	at org.dspace.core.Email.main(Email.java:523)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.dspace.app.launcher.ScriptLauncher.runOneCommand(ScriptLauncher.java:210)
	at org.dspace.app.launcher.ScriptLauncher.runOneCommand(ScriptLauncher.java:92)
	at org.dspace.app.launcher.ScriptLauncher.main(ScriptLauncher.java:80)
```

Reply:
Yes, the default template wasn't being loaded properly.  Fixed.